.bazelrc

```diff
@@ -71,6 +71,7 @@ common --experimental_allow_tags_propagation
 
 # Python
 common --@rules_python//python/config_settings:bootstrap_impl=script
+common --incompatible_default_to_explicit_init_py
```

Comment:
this shouldnt be necessary here - its only external repos (that use a mixture of our published and in-repo python deps) that need set it